src/index.js

```diff
@@ -164,7 +164,7 @@ class Cropper extends React.Component {
   }
 
   setNewZoom = zoom => {
-    const newZoom = Math.min(MAX_ZOOM, Math.max(zoom, MIN_ZOOM))
+    const newZoom = Math.min(this.props.minZoom, Math.max(zoom, this.props.maxZoom))
```

Comment:
you switched min and max here ;)

Reply:
Thanks to the e2e tests to detect this. 🙂

Reply:
Ah, sorry, will recommit